Resources/Audio/Voice/Talk/attributions.yml

```diff
@@ -2,3 +2,32 @@
   license: "CC-BY-4.0"
   copyright: "Recorded by https://github.com/PixelTheKermit"
   source: "https://github.com/space-wizards/space-station-14/pull/13945"
+
+- files: ["pai.ogg", "pai_ask.ogg", "pai_exclaim.ogg"]
+  license: "CC-BY-3.0"
+  copyright: "hubismal on GitHub"
+  source: "https://github.com/hubismal"
```

Comment:
If possible, the source field should point to a URL where the original file itself can be downloaded, not just the website/homepage of the creator. 

For files like this it should usually just be a permalink to the files on GH, though if all of these files were added in the same PR you can just link the PR.

Reply:
Actually after looking at https://github.com/space-wizards/RobustToolboxSpecifications/tree/master/RobustGenericAttribution it seems like the just linking the PR is fine, it doesn't need to be a permalink to the file itself.

Reply:
I've linked the commit itself, instead of the PR. I think that is more accurate.